client/settings/account-details/index.js

```diff
@@ -3,50 +3,69 @@ import { createInterpolateElement } from '@wordpress/element';
 import React from 'react';
 import SectionStatus from '../../components/section-status';
 import './style.scss';
+import { useAccount, useGetCapabilities } from 'wcstripe/data/account';
+
+const useIsCardPaymentsEnabled = () =>
+	useGetCapabilities().card_payments === 'active';
```

Comment:
This should probably be Yodaized:

```suggestion
	'active' === useGetCapabilities().card_payments;
```

Reply:
We can probably add an eslint rule if we'd like to enforce this 👍